docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.3"
+
+services:
+  app:
+    build: .
+    image: decentralize/peertube
+    environment:
+      NODE_CONFIG: '{"webserver": {"https": false, "port": 80}}'
+      PEERTUBE_HOSTNAME:
+      PEERTUBE_ADMIN_EMAIL:
+      PEERTUBE_SIGNUP_ENABLED:
+      PEERTUBE_TRANSCODING_ENABLED:
+    labels:
+      traefik.enable: "true"
+      traefik.frontend.rule: "Host:${PEERTUBE_HOSTNAME}"
+      traefik.docker.network: traefik_traefik
+      traefik.port: "9000"
```

Comment:
(see comment below) I think these can be removed too, in the non-swarm compose file.